WooCommerce/Classes/POS/Presentation/Item Selector/ChildItemList.swift

```diff
@@ -15,9 +15,9 @@ struct ChildItemList: View {
             .loading([])
     }
 
-    init(parentItem: POSItem, parentProduct: POSParentProduct) {
+    init(parentItem: POSItem, title: String) {
         self.parentItem = parentItem
-        self.parentProduct = parentProduct
+        self.title = title
```

Comment:
I much prefer this simplification – when there's so little to pass through, the view's more flexible if we just declare the title compared to passing a whole model object.

Reply:
Agreed!